File: A02/main.cpp
#include <iostream>
using namespace std;

int main() {
    int N, X, A[100];
    string ans = "No";
    cin >> N >> X;
    for(int i=0; i<N; i++){
        cin >> A[i];
        if(A[i] == X){
            ans = "Yes";
            break;
        }
    }
    cout << ans << endl;
    return 0;
}

File: A02/main.rs
use proconio::input;

fn main() {
    input! {
        N: i64,
        X: i64,
        A: [i64; N],
    }
    let mut ans= "No";
    for An in A {
        if An == X {
            ans = "Yes";
            break;
        }
    }
    println!("{}", ans);
}